enhancements/network/user-defined-network-segmentation.md

```diff
@@ -75,6 +76,8 @@ tenant to isolate traffic.
   are natively isolated from other tenants.
 * As a user, I want to be able to request a unique, primary network for my namespace without having to get administrator
   permission.
+* As a user, I want to be able to request new secondary networks for my namespace, without having to get administrator
```

Comment:
Since  creating NetworkAttachmentDefinitions (NAD) is possible by cluster-admin only, due to the possibility of a non-admin user creating a NAD that could break cluster network (like creating something on br-ex or an underlying bond), or worse - cause a disruption for cluster nodes as there is no restriction on the NAD spec one could specify bad spec that could exceed the node resources.

Support migrating from OpenStack to Kubenetes, where tenant (project admins) should be able to create tenant-network at the namespace they are managing.
OVN-K enable creating such networks but since it require creating NAD, only cluster admin can create such networks.

Enable sysadmin. of one org. to manage their project in Openshift and manage the network for VMs they are managing.
Similarly sysadmin of another org should be able to do the same for their own project (namespace they are managing)

In general, secondary networks can be used for, but not only, to segregate networks and manage which workloads could have communication.
For example: 
I have legacy app runs inside a VM (OCP virt), it should be accessible by a subset of workloads (pods).
I can achieve that by creating OVN overlay network, and connect the legacy app and permitted workloads to this network.